src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,33 @@
+package duke.tasks;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Date;
+
+public class Event extends Task {
+    protected LocalDateTime start;
+    protected LocalDateTime end;
+
+    public Event(String description, LocalDateTime start, LocalDateTime end) {
+        super(description);
+        this.start = start;
+        this.end = end;
+    }
+
+    /**
+     * Process Event to String to store in duke.txt
```

Comment:
I think we should use "Processes" here instead of "Process" for the method javadocs